### ISSUE TITLE
Can not be used with two OneToOneField

### ISSUE THREAD

**Issue by FUNNYDMAN** [daniyalzade/django_reverse_admin#36]
If I extend example form readme to:
```
inline_reverse = ['business_addr',
        ('home_addr', {'fields': ['street', 'city', 'state', 'zipcode']}),
        ('other_addr', {'fields': ['street', 'city', 'state', 'zipcode']}),
   ]
```
I get wrongly results: when I try to change two of these at the same time ony one of these is saved.

**Comment by daniyalzade** [comment]
@FUNNYDMAN this is fixed with PR #44

**Comment by daniyalzade** [comment]
@FUNNYDMAN available as of version - 2.7.0
